WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java

```diff
@@ -46,7 +46,7 @@ public void removeSite(String siteName) {
         clickOn(android.R.id.button1);
     }
 
-    public void startNewPost(String siteAddress) {
-        clickOn(R.id.fab_button);
+    public void startNewPost() {
+        clickOn("fab_button");
```

Comment:
@JavonDavis Can you explain why you made this change? I'm probably missing something 😄

Reply:
Right there's a known issue in Espresso's click that can cause it to long click some elements, because of that I created 

https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java#L98-L109

I'm unsure why it just started happening but the click started doing long clicks of the fab button 🤷‍♂

Reply:
Gotcha. Thanks!